ext/npm-extension.js

```diff
@@ -389,6 +389,25 @@ exports.addExtension = function(System){
 		oldConfig.apply(loader, arguments);
 	};
 
+	if (typeof steal !== undefined) {
+		steal.addNpmPackages = function(packages) {
+			packages = packages || [];
+
+			for (var pkg of packages) {
```

Comment:
![671107](https://cloud.githubusercontent.com/assets/724877/23322656/26b083d4-faa2-11e6-995e-76d34d8fcd60.gif)

working on it.